openedx/features/colaraz_features/api/views.py

```diff
@@ -169,3 +170,26 @@ def get(self, request, api_method, format=None):
             {'message': 'Job Alerts API is not enabled or is not configured properly'},
             status=status.HTTP_400_BAD_REQUEST
         )
+
+
+class CourseOutlineView(APIView):
+    authentication_classes = (OAuth2Authentication, )
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, course_id):
+        """
+        api to get course outline in sequence
+        :param request:
+        :param course_id:
+        :return: dictionary of course outline tree structure
+        """
+        user = request.user
+        try:
+            course_tree = get_course_outline_block_tree(request, course_id, user)
+        except:
```

Comment:
Can we except some specific exception here instead of this Broad exception handling?